docs/MAINTAINERS.md

```diff
@@ -20,6 +20,7 @@ To see what these roles do, and how to become one, look at [GOVERNANCE](./GOVERN
 - Noah Kennedy (@Noah-Kennedy)
 - dswij (@dswij)
 - tottoto (@tottoto)
+- Katelyn Martin (@cratelyn)
```

Comment:
```suggestion
- katelyn martin (@cratelyn)
```

small stylistic nit, i prefer to spell my name in lowercase letters :slightly_smiling_face: